### DISCUSSION TITLE
Board Tray

### DISCUSSION THREAD

**Discussion by bonz099** [xpDIY/3DPlacer#5] [General]
Is there a model for a board tray or fixture? If not, I can make one up.

**Comment by xpDIY** [comment]
For general fixture no, so far I make board specific fixture, and it is definitely good idea to make fixture or tray that can fit different sizes, so we can give it a go

**Reply by bonz099** [reply]
Can you add any existing board trays you may have? I can build off those. I'm still in the process of printing all my stuff, so I'm not sure how much height matters yet.

**Reply by xpDIY** [reply]
The PCB surface height is 43mm, I have pushed an example board fixture, and that is fixed one, which only fit for specific board I design, you can use that one as a reference.
